### ISSUE TITLE
[suggestion] Hint Feature

### ISSUE THREAD

**Issue by qowlz** [thomaspark/flexboxfroggy#223]
First, Sorry for my bad english.
As title says, I think this game will be better if add hint button to all level for those who keep submitting wrong answer.
The detailed description and sample image of the hint feature is as follows:

![image](https://user-images.githubusercontent.com/33592844/139005106-86c80656-e77c-4139-8a8c-866dac1a5608.png)
When player click hint button that looks like above image, show popup window that contains phrase that has a hint to solve easily.

In conclusion, I want to ask if my team can develop this feature.
Also, if there's any suggestion in this feature, please let me know.
Thank you.

**Comment by qowlz** [comment]
I thought about translating, First of all, how about temporarily put Google translation results for languages other than Korean and English? 
I think my team can translate Korean and English. 
I expect someone to contribute to other languages later.

**Comment by qowlz** [comment]
Hi @thomaspark I'm sorry I keep mentioning you. If you OK, please answer if I can either proceed as suggested above or not.

**Comment by thomaspark** [comment]
Hey @qowlz, I'd like to delay this feature for now, since it will introduce a lot of additional work in terms of designing good hints that I can't prioritize, plus making the translations.

**Comment by qowlz** [comment]
I understand. then, I'll close this issue for now.
